doc-Managing_Infrastructure_and_Inventory/topics/Instance_Evacuation.adoc

```diff
@@ -1,4 +1,4 @@
-If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (in the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
+If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (from the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
```

Comment:
@dayleparker Hey Dayle, thanks for your comment. I was meaning to bring this up in our meeting today - I have been pondering over this since I worked on this change recently. It's funny because I changed it to "in" when I first reviewed the guide (also committed the changes), but later changed it back to "from" - as documented by the original author - for the way the sentence is structured. :) However, I agree we should follow the convention that we have now decided on. I'll update the files soon.